### PR TITLE
Align pallet-fragment with new transfer in pallet-balances

### PR DESCRIPTION
**Pull request by alebaffa**
All `Cargo.toml` are just to make it work with the [other PR ](https://github.com/fragcolor-xyz/substrate/pull/5) on Substrate . I will fix them once that PR is merged and Substrate retagged.

Unfortunately, I needed to specify `fungible` and `fungibles` for all calls to `Inspect` and `Transfer` because the two `transfer()` in `pallet_assets` and `pallet_balances` use `fungibles` and `fungibles`, respectively.

### CODE REVIEW

**Comment by alebaffa** [comment]
@sinkingsugar putting you in review, even if this is a draft PR, to know what you think of it.

File: pallets/fragments/src/lib.rs [hunk 1]
```diff
@@ -759,12 +759,12 @@ pub mod pallet {
 				)
 				.map_err(|_| Error::<T>::InsufficientBalance)?;
 			} else {
-				<pallet_balances::Pallet<T> as Currency<T::AccountId>>::transfer(
+				<pallet_balances::Pallet<T> as fungible::Transfer<T::AccountId>>::transfer(
```
Comment (sinkingsugar) [inline pallets/fragments/src/lib.rs]: confused on this, balances does not implement fungibles afaik? if it does we need to prevent transfer there too

Comment (alebaffa) [reply pallets/fragments/src/lib.rs]: It is blocked now on Substrate. In the last two commits [here](https://github.com/fragcolor-xyz/clamor/pull/166/commits/a2098fc6b36d3157dd012e1b0b1dd8f2c54ecb34) and [here](https://github.com/fragcolor-xyz/clamor/pull/166/commits/a380f993efb11a3d60eb17f65ee747878f0825db) I call `do_transfer` from `pallet_balances`, which is now public and I also removed the use of `fungible` trait here.

**Review by sinkingsugar — APPROVED** [review]

**Comment by alebaffa** [comment]
@sinkingsugar can you approve the [PR ](https://github.com/fragcolor-xyz/substrate/pull/5)on Substrate too? So, that I will clean that first (squash commits), retag with `fragnova-v0.0.6` and use it here.

**Pull request merged**